libs/ui/packages/native/src/components/Layout/List/VerticalStepper/index.tsx

```diff
@@ -28,6 +28,4 @@ export default function VerticalStepper({ steps, onTapIndex, nested, ...props }:
       ))}
     </Flex>
   );
-}
-
-VerticalStepper.ItemStatus = ItemStatus;
+});
```

Comment:
I know that the live-hub team likes doing this: it's scoping the `ItemStatus` to the component `VerticalStepper` -> but i don't really know how i feel about it, it's adding a prop that is not part of a React component + it could make sense if `ItemStatus` was also a component, but here it's an enum, so i agree with you

Reply:
I actually had to do it, I can't remember exactly the reason, but it was necessary for me to import somewhere else.